README.md

```diff
@@ -206,17 +206,20 @@ GPU, much faster than previous methods, while the generated instances can be dir
 - [WebGL Viewer 2](https://github.com/cvlab-epfl/gaussian-splatting-web)
 - [Three.js](https://github.com/mkkellogg/GaussianSplats3D)
 - [A-Frame](https://github.com/quadjr/aframe-gaussian-splatting)
+- [gsplat.tech](https://gsplat.tech/)
```

Comment:
I know this resource but I think there is no code available? At least I did not find any or maybe I overlooked it.

Reply:
It's not open source, feel free to remove gsplat.tech if you only want to reference open source viewers :)

Reply:
I've rebased and removed gsplat, feel free to merge if you're happy with it